components/scream/src/control/surface_coupling.cpp

```diff
@@ -12,30 +13,51 @@ SurfaceCoupling (const field_mgr_ptr& field_mgr)
 {
   auto grid = m_field_mgr->get_grid();
 
-  m_num_cols  = grid->get_num_local_dofs();
+  m_num_cols = grid->get_num_local_dofs();
+  m_num_levs = grid->get_num_vertical_levels();
+
+  using namespace ShortFieldTagsNames;
+  using namespace ekat::units;
+
+  FieldLayout layout { {COL}, {m_num_cols} };
+  Units nondim(0,0,0,0,0,0,0);
+  auto grid_name = grid->name();
+  FieldIdentifier id ("dummy_field", layout, nondim, grid_name);
+  dummy_field = Field<const double>(id);
 
   EKAT_REQUIRE_MSG(grid->type()==GridType::Point,
       "Error! Surface coupling only implemented for 'Point' grids.\n"
       "       Input grid type: " + e2str(grid->type()) + "\n");
 }
 
 void SurfaceCoupling::
-set_num_fields (const int num_imports, const int num_exports)
+set_num_fields (const int num_total_imports, const int num_scream_imports, const int num_exports)
```

Comment:
Another subjective one: 'total' is a very overloaded word; we may call the first one `num_cpl_imports`, which emphasizes it's the number of imports offered by the coupler.